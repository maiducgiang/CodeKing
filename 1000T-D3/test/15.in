81
92843 78875 69662 34668
23904 41476 81336 57371
35841 90283 39747 21390
38402 80286 41928 24133
27519 41370 60765 14737
43171 41544 28181 39104
33101 23918 52140 19757
75532 87076 40176 62050
41063 1049 84813 67985
99780 60348 68293 34277
97661 61199 98756 86712
39529 96852 96527 87406
52361 69801 12153 71426
81041 30753 6709 64771
31600 88112 31092 36344
51926 49479 17127 53079
32622 17868 19611 61891
25496 39066 2855 83179
50035 47524 79220 12792
67878 45599 94175 67549
18812 88931 29227 55409
98301 80855 47778 86867
84197 7933 91220 8452
90700 43582 41075 24202
51820 85063 33067 35280
19257 2998 13224 11265
95785 17468 83956 54066
92617 23873 64082 70027
56880 55620 58128 44761
14668 64335 40832 34231
10685 30265 56473 32462
17287 92195 6185 58426
25173 16750 11017 51294
86685 96301 23138 37577
88924 97824 29401 94107
60526 56466 49835 12905
60132 91638 78012 9681
97569 51639 23296 38718
7295 93262 57859 31258
19192 1547 45910 49549
1013 34422 1943 28536
55861 90731 1546 63825
98914 98207 5936 45436
65230 82674 73073 42167
12337 36831 21756 14380
83396 79370 5222 3680
60069 91459 17544 20510
24351 96279 33177 7677
79047 72306 40596 44269
96396 41160 13801 98316
51379 68623 27519 25351
87525 89585 43584 97023
19857 42144 70941 12186
85199 85842 55929 40512
67903 29822 23125 7566
84808 85256 60582 98326
10953 5751 33469 68353
68721 85096 8891 27775
87014 10170 25999 15311
6411 80923 5029 56113
40655 71921 72585 5267
46541 91348 83515 60344
91087 43485 51735 86920
40853 36287 5107 24683
6333 95503 52953 37048
82169 23501 7015 57466
83402 78742 1319 40826
10956 19571 10914 56885
13371 7447 46707 82044
12267 42825 56831 14915
8534 2219 46130 32399
81130 87193 93347 9232
93789 22228 18277 7026
59041 83865 69257 95682
89584 32503 67946 10824
51791 35004 33454 16813
71756 93853 66245 14938
6775 66313 81023 1705
30785 55158 66546 88796
58082 39884 73500 23923
95107 36083 27876 48711
