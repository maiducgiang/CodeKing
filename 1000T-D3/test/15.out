3707477316
2800632576
2930545164
2682867818
1247203518
1670387063
951349546
5150552344
3465023647
4761477972
8907528960
8988288995
1484634786
750025946
2905541504
1736763435
990159390
184324510
806848428
5178291676
3091722121
7857166273
7023851796
2990143414
2927455881
248781486
7519606940
6077007579
2562848100
2530103252
1452597810
677146170
461865691
3479857253
5490577948
931023830
3039983964
3069344082
5201991763
886734253
68850205
226631232
1170110256
4025238822
710297524
313105372
1782721350
2573297883
3474571950
1898410356
1777046665
7719162240
1823039469
3487544235
1106256377
8365972144
559066176
1367586947
2153580026
439202686
2483572495
6550526709
6242405670
393953980
4956443994
741275050
213867936
333171678
887673936
1919604003
363204331
1314955021
1464094325
7368396730
4230238098
1150114550
2535706893
4835498749
4355510518
2291697932
3402992037
